Generals/Code/GameEngine/Source/GameClient/GUI/GUICallbacks/Menus/SkirmishGameOptionsMenu.cpp

```diff
@@ -778,22 +778,33 @@ void updateMapStartSpots( GameInfo *myGame, GameWindow *buttonMapStartPositions[
 	if (it == TheMapCache->end())
 	{
 		for (Int i = 0; i < MAX_SLOTS; ++i)
-			buttonMapStartPositions[i]->winHide(TRUE);
+    {
+      if ( buttonMapStartPositions[i] != NULL )
+      {
+  			buttonMapStartPositions[i]->winHide(TRUE);
+      }
+    }
```

Comment:
Indentation in this file looks wrong.

Reply:
Nevermind, the whitespacing here just has tabs and spaces mixed. It is like that in Zero Hour.